numpy/random/_generator.pyx

```diff
@@ -4571,12 +4571,24 @@ cdef class Generator:
         -------
         None
 
+        .. note::
+            An important distinction between methods ``shuffle``  and ``permuted`` is 
+            how they both treat the ``axis`` parameter which can be found at 
+            :ref:`generator-handling-the-axis-parameter`.
+
+        See Also
+        --------
+        permuted
+        permutation
+
         Examples
         --------
         >>> rng = np.random.default_rng()
         >>> arr = np.arange(10)
         >>> rng.shuffle(arr)
         >>> arr
+        [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
Did you mean to put this before the `rng.shuffle(arr)`?

Reply:
Be sure that the output matches. `arr` is not a list, so it won't look like this.

Reply:
It is this way on the Official DOCS as well. [numpy.random.generator](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.shuffle.html).

But I guess this is what this PR is for(To fix it and Add new things)